src/test/java/jenkins/plugins/office365connector/TimeUtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package jenkins.plugins.office365connector;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+/**
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public final class TimeUtilsTest {
+
+    @Test
+    public void dateToString_ReturnsDateAsString() {
```

Comment:
General comment:
my plan is to provide tests for each functionality that goes out of the ``Ofice365ConnectorWebhookNotifier`` so let me know if this is OK to you

Reply:
That is awesome. Thanks a ton... :)